### PR TITLE
Fix <pre> styling in lists

### LINKED ISSUE #6552: Project description: rendering of <pre> in lists ugly
Given the following HTML structure in a project description:

Screenshot from 2019-08-30 07-29-40

The rendering is poor:

Screenshot from 2019-08-30 07-29-36

This requires updating warehouse/static/sass/blocks/_project-description.scss

---

**Screenshot Required**: This issue will require an update to the visual design of the site. To help our team give you faster feedback, include a screenshot in your Pull Request.

### LINKED ISSUE #10242: Maybe Bug ))
Top Image
https://pypi.org/project/django-admin-autocomplete-filter/

### PR DESCRIPTION
**Pull request by tusharsadhwani**
Resolves #6552, fixes #10242.

Before:
image

After:
image

### CODE REVIEW

**Comment by tusharsadhwani** [comment]
Btw, screenshots taken from https://pypi.org/project/twine/1.14.0/

File: warehouse/static/sass/blocks/_project-description.scss [hunk 1]
```diff
@@ -129,6 +125,13 @@
     padding: 2px 0;
   }
 
+  li p:first-child {
+    margin-top: 0;
+  }
```
Comment (tusharsadhwani) [inline warehouse/static/sass/blocks/_project-description.scss:130]:
```suggestion
  }

```

**Review by ewjoachim — APPROVED** [review]

**Comment by tusharsadhwani** [comment]
Added another little fix for a scrolling issue.
Before:
image

After:
image

Tested on [django-htmx](https://pypi.org/project/django-htmx/)

**Comment by tusharsadhwani** [comment]
Ran into this css issue again today. Text overflow:
image

@di bump?

**Review by di — APPROVED** [review]

**Pull request merged**